polaris-react/src/components/Popover/components/Pane/Pane.tsx

```diff
@@ -47,9 +53,12 @@ export function Pane({
   const content = sectioned
     ? wrapWithComponent(children, Section, {})
     : children;
-  const style = height
-    ? {height, maxHeight: height, minHeight: height}
-    : undefined;
+
+  const style = {
+    height: height || undefined,
+    maxHeight: maxHeight || undefined,
+    minHeight: minHeight || undefined,
+  };
```

Comment:
I think this can be cleaned up a bit since these height values are either a value or undefined by default

```suggestion
  const style = {height: height, maxHeight: maxHeight, minHeight: minHeight};
```